src/test/resources/contingencies/6noeuds.json

```diff
@@ -1 +1 @@
-{"type":"default","version":"1.0","name":"contingencies","contingencies":[{"id":"SE_G","elements":[{"id":"SE_G","type":"GENERATOR"}]},{"id":"SE_L2","elements":[{"id":"SE_L2","type":"LOAD"}]},{"id":"SO_G2","elements":[{"id":"SO_G2","type":"GENERATOR"}]},{"id":"SO_G1","elements":[{"id":"SO_G1","type":"GENERATOR"}]},{"id":"N_G","elements":[{"id":"N_G","type":"GENERATOR"}]},{"id":"HVDC1","elements":[{"id":"HVDC1","type":"HVDC_LINE"}]},{"id":"SO_NO_1","elements":[{"id":"SO_NO_1","type":"BRANCH"}]},{"id":"SO_NO_2","elements":[{"id":"SO_NO_2","type":"BRANCH"}]}]}
\ No newline at end of file
+{"type":"default","version":"1.0","name":"contingencies","contingencies":[{"id":"SE_L2","elements":[{"id":"SE_L2","type":"LOAD"}]},{"id":"SO_NO_2","elements":[{"id":"SO_NO_2","type":"BRANCH"}]},{"id":"SO_NO_1","elements":[{"id":"SO_NO_1","type":"BRANCH"}]},{"id":"N_G","elements":[{"id":"N_G","type":"GENERATOR"}]},{"id":"SO_G2","elements":[{"id":"SO_G2","type":"GENERATOR"}]},{"id":"HVDC1","elements":[{"id":"HVDC1","type":"HVDC_LINE"}]},{"id":"SO_G1","elements":[{"id":"SO_G1","type":"GENERATOR"}]},{"id":"SE_G","elements":[{"id":"SE_G","type":"GENERATOR"}]}]}
```

Comment:
Why the order has changed?

Reply:
There was in fact a random process regenerating the contingencies in the test case creator. I avoided using that to regenerate the reference as I was regenerating different tests cases. This is fixed.